root/static/scripts/release-editor/fields.js

```diff
@@ -498,22 +498,22 @@ class Medium {
       return self.tracksUnknownToUser() ||
              self.tracks().every(t => t.hasTitle());
     });
-    this.hasVariousArtistTracks = ko.computed(function () {
+    this.hasVariousArtistsTracks = ko.computed(function () {
```

Comment:
This one feels debatable; see e.g. https://english.stackexchange.com/questions/497087/can-plural-noun-be-followed-by-another-plural-noun. Since this isn't user-facing, I figure it's probably marginally better for maintainability to use "VariousArtists" here rather than the singular form.